daemon/cmd/local_node_sync.go

```diff
@@ -54,11 +55,30 @@ func (ini *localNodeInitializer) InitLocalNode(ctx context.Context, n *node.Loca
 	return nil
 }
 
-func newLocalNodeInitializer(p localNodeInitializerParams) node.LocalNodeInitializer {
-	return &localNodeInitializer{p}
+func (ini *localNodeSynchronizer) SyncLocalNode(ctx context.Context, store *node.LocalNodeStore) {
+	if ini.LocalNode == nil {
+		return
+	}
+
+	for ev := range ini.LocalNode.Events(ctx) {
```

Comment:
Hmm wonder if we want to be defensive here and not update too often since changes to the local node kick off bunch of work? could use `stream.Debounce` and the `Observable` of `Resource[*Node]`.

OTOH, not sure whether that should instead be the responsibility of `LocalNodeStore` or the individual subscribers (which would know the cost of the work and what debouncing durations to use...)..